content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -44,8 +44,7 @@ management policies to determine some placement preferences on the node.
 
 ### Configuration
 
-The CPU Manager is an alpha feature in Kubernetes v1.8. It was enabled by
-default as a beta feature since v1.10.
+The CPU Manager feature is currently in beta and is enabled by default.
```

Comment:
@miteshskj would you be OK mentioning that this it's in beta since v1.10 as part of the feature-state shortcode on line 12?

If so, you might be able to omit this mention that it's enabled by default and in beta; the shortcode covers that.

Reply:
@sftim  - I have seen that the feature state is added to a heading or a title for a section. Since this is mentioned as a statement in a para, how do you suggest I add the feature-state shortcode?  
=======
### Configuration

CPU Manager - FEATURE STATE: Kubernetes v1.10 beta
============
Does this work ?

Reply:
Tried adding the feature-state, looks out of place - 
![Screenshot from 2019-10-31 23:09:06](https://user-images.githubusercontent.com/47820816/67972198-36d5f000-fc34-11e9-9759-16a79a9d8c88.png)

Reply:
@miteshskj - tweak the shortcode on line 12 to mention what version it went into beta. Then you can remove “The CPU Manager feature is currently in beta and is enabled by default.“

Reply:
Maybe also change:
```markdown
* `none`: the default, which represents the existing scheduling behavior.
```

because “existing” is misleading now. I suggest:
```markdown
* [`none`](#none-policy): the default policy.
* [`static`](#static-policy): allows pods with certain resource characteristics to be
```

Reply:
@sftim   -Added the changes.